cogs/group_subject_linking.py

```diff
@@ -96,14 +105,17 @@ async def add(self, ctx: Context,
             verb = "updated"
         else:
             await self.db.insert_one((study_role, subject_role, default))
-            study_role = "linked"
+            verb = "linked"
         embed = Embed(title="Linking Add",
-                      description=f"Successfully {study_role} {study_role.mention} to {subject_role.mention} "
+                      description=f"Successfully {verb} {study_role} {study_role.mention} to {subject_role.mention} "
```

Comment:
remove "{study_role}"
Copy paste this since you don't seem to manage it:
`description=f"Successfully {verb} {study_role.mention} to {subject_role.mention} "`